shaders/src/main/glsl/samples/310es/prefix_sum_checkers.frag

```diff
@@ -0,0 +1,95 @@
+#version 310 es
+
+/*
+ * Copyright 2019 The GraphicsFuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+precision highp float;
+
+layout(location = 0) out vec4 _GLF_color;
+
+uniform vec2 injectionSwitch;
+
+uniform vec2 resolution;
+
+vec2 pattern(in vec2 x){
+	vec2 n = floor(x);
+	vec3 m = vec3(1.);
+	
+    for (int j = -1; j <= int(injectionSwitch.y); j++) {
+		for (int i = -1; i <= int(injectionSwitch.y); i++) {
```

Comment:
Indentation above.

Reply:
The indentation is now using spaces instead of tabs.